utilities/ideascale-importer/ideascale_importer/ideascale/importer.py

```diff
@@ -263,6 +252,20 @@ def __init__(
             except Exception as e:
                 raise ReadProposalsScoresCsv(repr(e)) from e
 
+    async def load_config(self):
+        """Load the configuration setting from the event db."""
+
+        logger.debug("Loading ideascale config from the event-db")
+
+        config = ideascale_importer.db.models.Config(row_id=0, id="ideascale", id2="10", id3="", value=None)
```

Comment:
Are you sure ?
So actually into the `id2` should be put an event_id value (I have made a mistake, fixed it) and it is not corresponded to the this config row_id

Reply:
I mean, this was the way I intended for us to use that. We'll need IdeaScale parameters for each event so I thought it would be a good ideia to have params for each event.

Reply:
Ohh, I got what do you mean, sorry for misunderstanding.
I have put into `id2` row_id of an event already, here in the code it was a mistake I have hardcoded this value, already fixed, now it is taken from `self.event_id`.